src/hotspot/share/memory/virtualspace.hpp

```diff
@@ -35,19 +35,20 @@ class outputStream;
 class ReservedSpace {
   friend class VMStructs;
  protected:
-  char*  _base;
-  size_t _size;
-  size_t _noaccess_prefix;
-  size_t _alignment;
-  size_t _page_size;
-  bool   _special;
-  int    _fd_for_heap;
+  char*    _base;
+  size_t   _size;
+  size_t   _noaccess_prefix;
+  size_t   _alignment;
+  size_t   _page_size;
+  bool     _special;
+  int      _fd_for_heap;
+  bool     _executable;
+  MEMFLAGS _nmt_flag;
```

Comment:
See my remark below. This member, and probably others (e.g. page size and size) could and should probably be const. Food for follow up PRs.

Reply:
The getter method made as `const`.